src/views/bibliography_item.ts

```diff
@@ -80,6 +87,27 @@ export class BibliographyItemView<PropsType extends BaseNodeProps>
       try {
         const fragment = sanitize(bibliography.outerHTML)
         this.dom.appendChild(fragment)
+        //
+        // const doubleButton = document.createElement('div')
```

Comment:
Why is this commented out?

Reply:
I don't know what happened with git. please ignore it, until we merge [PR#7](https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/7), I will open a new PR.